pkg/runner/runner.go

```diff
@@ -170,7 +172,15 @@ func (runner *runnerImpl) NewPlanExecutor(plan *model.Plan) common.Executor {
 				}
 				pipeline = append(pipeline, common.NewParallelExecutor(maxParallel, stageExecutor...))
 			}
-			return common.NewParallelExecutor(runtime.NumCPU(), pipeline...)(ctx)
+			var ncpu int
+			info, err := container.GetHostInfo(ctx)
+			if err == nil {
```

Comment:
This doesn't seem to be right
```suggestion
			if err != nil {
```

Reply:
right, typo